doc/material-dsp.md

```diff
@@ -32,6 +32,25 @@ This package will be automatically present in the project after [creating a new
 * The package is already present when you select _Material_ theme during project creation:
    ![Selection of Material theme when creating a project using the Uno Template Wizard](assets/material-theme-selection-wizard.png)
 
+It is possible to configure the import process. The _UnoDspImportColors_ item found in the _csproj_ file has a number of attributes we can set:
+
+| Attribute      | Description                     | Default Value           |
+|----------------|---------------------------------|-------------------------|
+| `Include`      | Style files to input            |                         |
+| `Generator`*   | Type of generator to use        | `xaml`                  |
+| `OutputPath`   | Path to use for the output      | Input file directory    |
```

Comment:
are there any restrictions on the Include or OutputPath values? eg: can the OutputPath be Styles/json_file.g.cs ?